### DISCUSSION TITLE
Is there a resource_path equivalent?

### DISCUSSION THREAD

**Discussion by bilogic** [laravel-zero/laravel-zero#369] [Q&A] [answered]
Hi,

I was wondering if there is a `resources` equivalent that gets bundled into the standalone `phar` as well?
I'm trying to create a tool to generate config files etc, looking for a place to store the stub files, which I want included in the `phar` eventually.

Thank you.

**Comment by bilogic** [comment]
What I did was to create a `resources` class and used heredocs to return content of the various files. Works at the moment, but even better if there was an official way. Thank you.

**Comment by owenvoke** [comment, marked as answer]
The [`resource_path()` function](https://github.com/laravel-zero/foundation/blob/main/src/Illuminate/Foundation/helpers.php#L694-L705) should exist, and will point to the `resources/` directory. However, you'll need to create this and add it to your `box.json` directories list.

**Reply by bilogic** [reply]
Ok, let me read up on `box.json`. Thank you.
